.travis.yml

```diff
@@ -0,0 +1,43 @@
+language: php
+
+php:
+  - 5.5
+  - 5.6
+
+env:
+  - TYPO3_BRANCH=TYPO3_6-2
+  - TYPO3_BRANCH=TYPO3_7-0
```

Comment:
7.0 has been removed from Travis in the next patch. Only latest 7.x stable and 6.2 of course is kept for Travis.